### PR TITLE
update UPGRADE.md

### PR DESCRIPTION
**Pull request by pweil-**
Updates to include info about priority.  This does not specifically mention the reconcile command since that PR is not merged however it points to the openshift-docs section that will contain the information about the command.  I will remove the info from the openshift-docs PR if it does not make it.

Docs PR: https://github.com/openshift/openshift-docs/pull/1078

@openshift/team-project-committers
@liggitt

### CODE REVIEW

**Comment by smarterclayton** [comment]
Approved, but not reviewed.

File: UPGRADE.md [hunk 1]
```diff
@@ -32,10 +32,10 @@ At that time, the openshift docker registry image must be upgraded in order to c
 
 1. The `volume.metadata` field is deprecated as of Origin 1.0.6 in favor of `volume.downwardAPI`.
 
-1. New fields (`fsGroup`, `supplementalGroups`, `allowHostPID` and `allowHostIPC`) have been added 
-to the default SCCs in Origin 1.0.7.  These allow you to control groups for persistent volumes,
-supplemental groups for the container, and usage of the host PID/IPC namespaces.  The fields will 
-default as follows for existing SCCs:
+1. New SCCs and additional fields on SCCs have been added in Origin 1.0.7.  To pick up the new SCCs
```
Comment (smarterclayton) [inline UPGRADE.md]: 1.0.8

Comment (pweil-) [inline UPGRADE.md]: fixed

**Comment by smarterclayton** [comment]
LGTM [merge] so it's in 1.1

**Pull request merged**